limitador/src/storage/distributed/mod.rs

```diff
@@ -279,6 +296,49 @@ impl CrInMemoryStorage {
     }
 }
 
+async fn process_re_sync(
```

Comment:
Is the contract that this gets called _prior_ receiving any remote updates?
Not that it would matter in terms of correctness, but... see [below](https://github.com/Kuadrant/limitador/pull/343/files#diff-c3ae63bee7abe7f0e5a1c02891d517102a556f6855d8df57c7c654f29a3c05efR316)

Reply:
Ok, no... I misread this, this _sends_ the data to a newly connected peer! right?

Reply:
correct..  also note that we only re-sync 1 peer at a time since the requests to resync the peer are delivered on a channel service by 1 task... not sure if that's good or not.  I was thinking that you might want to avoid impacting this node too much by other peers connecting to it.  Maybe what would be ideal is resyncing up to X concurrent peers.

Reply:
👌 let's iterate again, we'll test these things out in "production" scenarios and improvise from actual data.